src/ecmult_impl.h

```diff
@@ -302,6 +377,17 @@ static void secp256k1_ecmult_context_init(secp256k1_ecmult_context *ctx) {
 }
 
 static void secp256k1_ecmult_context_build(secp256k1_ecmult_context *ctx, const secp256k1_callback *cb) {
+#if ECMULT_WINDOW_SIZE <= ECMULT_STATIC_TABLE_MAX
+    (void)cb;
+    if (ctx->pre_g != NULL) {
+        return;
+    }
+    ctx->pre_g = (secp256k1_ge_storage (*)[ECMULT_TABLE_SIZE(WINDOW_G)])secp256k1_pre_g_static_context;
+#ifdef USE_ENDOMORPISM
+    ctx->pre_g_128 = (secp256k1_ge_storage (*)[ECMULT_TABLE_SIZE(WINDOW_G)])secp256k1_pre_g_128_static_context;
+#endif
+    return;
```

Comment:
This explicit return seems unnecessary as the rest of the `#ifdef` already covers the entire function.

Reply:
fixed